### DISCUSSION TITLE
Where should I place my Frog?

### DISCUSSION THREAD

**Discussion by Rabscuttler** [Ribbit-Network/ribbit-network-frog-hardware#54] [General]
Where is everyone placing their sensors?

I'd love to know what folks think is best. I'm wandering about adverse impacts from siting to near to gas boiler flues and I also have an outdoor pizza oven that burns gas - I'd assume away from these things would be better.

I could site it on an exterior wall on the 1st floor (one above the ground floor..) - would that be preferable to a ground floor exterior wall? 

The power cable brings a minor challenge as well, I've ordered a USB C extension cable, though it might have been simpler to use a USB A extension cable and an adapter, not sure.

To make this more concrete, I've marked a few potential locations in the below images with blue stars, and highlighted potential unwanted CO2 sources in red. Which spot is best - 1,2,3,4 or 5?

Photos of your installations welcome :)

![IMG_5325](https://user-images.githubusercontent.com/1125376/137331894-5bea763a-9c19-43ea-b4bd-0b8b0914306f.jpg)
![IMG_5326](https://user-images.githubusercontent.com/1125376/137331901-fd2a9bb7-112e-490d-b00a-e099e2bd0432.jpg)

**Comment by keenanjohnson** [comment]
Hey there! I just saw this discussion! So sorry for not responding sooner! Yes I would also assume that away from the emissions sources would be the best. Looking at your photos, it seems to me that location 4 is possibly the furthest away from your emission sources, so that would probably be the best @Rabscuttler!
